### PR TITLE
Sticky search

### PR DESCRIPTION
**Pull request by Siddharth11**
Closes: #173 

## Demo
sticky

### CODE REVIEW

File: src/MapsWithSearch/Input/index.js [hunk 1]
```diff
@@ -167,49 +168,55 @@ class LocationSearchInput extends React.Component {
...
+            className="places-autocomplete-wrapper"
+          >
+            <div>
+              <InputGroup className="location-search-group">
+                <Form.Control
+                  {...getInputProps({
+                    placeholder: this.props.translations
+                      .location_search_placeholder,
+                    defaultValue: this.props.defaultValue,
+                    className: "location-search-input rounded-0",
```
Comment (Siddharth11) [inline src/MapsWithSearch/Input/index.js:182]: added `rounded-0` to remove border-radius. looks weird on sticky elements

**Comment by asanjeevrao** [comment]
experience looks good @Siddharth11. what happens when someone scrolls and then makes a new search? guessing they are reset to the top of the page again? not able to test this because autocomplete is not plugged in

**Comment by arjun27** [comment]
not exactly sure why the autocompletion is broken in the default state of the search bar. the sticky version works. seems like a css issue.

**Comment by Siddharth11** [comment]
> not exactly sure why the autocompletion is broken in the default state of the search bar. the sticky version works. seems like a css issue.

@arjun27 this is fixed now. you can verify it on preview.

**Pull request merged**